decoder.go

```diff
@@ -37,18 +37,17 @@ int get_video_codec(AVFormatContext *avFormatCtx) {
         if (codecParams->codec_id == AV_CODEC_ID_H264) {
             return AV_CODEC_ID_H264;
         } else if (codecParams->codec_id == AV_CODEC_ID_H265) {
-            found_h265 = 1;
+            return AV_CODEC_ID_H265;
```

Comment:
This may not be prioritized correctly because you could in theory have multiple tracks for a given endpoint in any order. So if returning inside the track iterator here, you may never see the higher priority codec. In practice, I doubt this would come up very often as you normally would only have one video codec for an endpoint.

Reply:
Discussed offline: we will leave the incorrect prioritization in this PR where @seanavery will pick the logic up and refactor it